term_posix.go

```diff
@@ -41,9 +43,11 @@ func (t *Term) SetCbreak() error {
 // CBreakMode places the terminal into cbreak mode.
 func CBreakMode(t *Term) error {
 	var a attr
-	if err := termios.Tcgetattr(uintptr(t.fd), (*syscall.Termios)(&a)); err != nil {
+	term, err := termios.Tcgetattr(uintptr(t.fd))
+	if err != nil {
 		return err
 	}
+	a = *(*attr)(term)
```

Comment:
Thanks. What about all the rest of them in that file ?

On Thu, Feb 11, 2016 at 4:31 PM, AZ Huang <notifications@github.com> wrote:

> In term_posix.go
> <https://github.com/pkg/term/pull/16#discussion_r52565603>:
>
> >  		return err
> >  	}
> > +	a = *(*attr)(term)
>
> Fixed. Should be clearer now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkg/term/pull/16/files#r52565603>.
>